src/pages/LoginSignupPage/SignupPage.tsx

```diff
@@ -0,0 +1,12 @@
+import Signup from '@components/LoginSignupPage/Signup';
+import { Stack } from '@mui/material';
+
+const SignupPage = (): JSX.Element => {
+  return (
+    <Stack>
+      <Signup />
+    </Stack>
```

Comment:
```suggestion
    <Signup />
```

여기도 불필요하다면 태그 중첩 없애줘도 될 것 같아보입니다!